src/pages/change-password/index.js

```diff
@@ -0,0 +1,11 @@
+import { ImmediateWindow } from 'src/windows'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import ChangePassword from 'src/components/Shared/ChangePassword'
+
+const ChangePass = () => {
+  return (
+    <ImmediateWindow datasetId={ResourceIds.ChangePassword} titleName={'ChangePassword'} Component={ChangePassword} />
```

Comment:
title should be a label no?

Reply:
this is a label, in the ImmediateWindow we have title: _labels[titleName]